src/osd/OSD.cc

```diff
@@ -9923,35 +10028,34 @@ void OSD::ShardedOpWQ::_process(uint32_t thread_index, heartbeat_handle_d *hb)
 
   // [lookup +] lock pg (if we have it)
   if (!pg) {
-    pg = osd->_lookup_lock_pg(item.first);
+    pg = osd->_lookup_lock_pg(token);
   } else {
     pg->lock();
   }
 
```

Comment:
@tchaikov you defined get_order_locker(). but osd->_lookup_lock_pg()(only for the pg lock) is still being used. How can we generalize this?